docs/versioned_docs/version-zenith/user-guide/cloud/snippets/get-started/ci/actions.yml

```diff
@@ -0,0 +1,27 @@
+name: dagger
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      -
+        name: Call Dagger Function
+        uses: dagger/dagger-for-github@v5
+        with:
+          version: "0.9.11"
+          verb: call
+          module: github.com/kpenfound/dagger-modules/golang@v0.1.5
+          args: build --project=. --args=./cmd/dagger
+        // highlight-start
+          cloud-token: ${{ secrets.DAGGER_CLOUD_TOKEN }}
+      # for ephemeral runners only: override the default docker stop timeout and
+      # give the Dagger Engine more time to push cache data to Dagger Cloud
+      - name: Stop Engine
+        run: docker stop -t 300 $(docker ps --filter name="dagger-engine-*" -q)
```

Comment:
error in GHA
```
Run docker stop -t 300 $(docker ps --filter name="dagger-engine-*" -q)
  docker stop -t [3](https://github.com/vikram-dagger/example/actions/runs/8053867613/job/21997182892#step:4:3)00 $(docker ps --filter name="dagger-engine-*" -q)
  shell: /usr/bin/bash -e {0}
"docker stop" requires at least 1 argument.
See 'docker stop --help'.
```

Reply:
Thanks. I've made the change in https://github.com/dagger/dagger/pull/6740/commits/0c2f2a18a0c2a37a33ca40adbd4776c6ec9f7cfc, also removed the note for the GHA section. Let me know in case further changes are needed for this.